benchmark/_cli.js

```diff
@@ -39,13 +42,9 @@ function CLI(usage, settings) {
     } else if (mode === 'both' && arg[0] === '-') {
       // Optional arguments declaration
 
-      if (arg[1] === '-') {
-        currentOptional = arg.slice(2);
-      } else {
-        currentOptional = arg.slice(1);
-      }
+      currentOptional = arg.split('-').pop();
```

Comment:
That's incorrect, if the flag is e.g. `--port-number`, it would return `number` instead of `port-number`. Also using `.split()` requires more work than calling `.slice()`
```suggestion
      currentOptional = arg.slice(arg[1] === '-' ? 2 : 1);
```

Reply:
LGTM, I missed this case, and I will add this case to test cases